protoc_plugin/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 15.0.4
```

Comment:
I think this needs a merge already.

Also there needs to be a corresponding update in `protoc_plugin/pubspec.yaml`